content/blog/understanding-the-dom/index.md

```diff
@@ -0,0 +1,519 @@
+---
+{
+	title: 'Understanding The DOM: How Browsers Show Content On-Screen',
+	description: 'Learn how the browser internally handles HTML and CSS to show the user webpages on-screen',
+	published: '2019-11-26T22:12:03.284Z',
+	authors: ['crutchcorn'],
+	tags: ['dom', 'browser internals'],
+	attached: [],
+	license: 'cc-by-nc-sa-4'
+}
+---
+
+Any web application relies on some fundamental technologies: HTML, CSS, and JavaScript. Even advanced front-end JavaScript frameworks such as Angular, React, or Vue will utilize some level of HTML to load the JavaScript. That said, how the browser handles HTML and CSS under-the-hood can be quite the mystery. In this article, I'm going to explain what the browser does to understand what it should show to the user.
```

Comment:
```suggestion
Every web application relies on some fundamental technologies: HTML, CSS, and JavaScript. Even advanced front-end JavaScript frameworks such as Angular, React, or Vue still utilize some amount of HTML to load the JavaScript. That said, how the browser handles HTML and CSS under-the-hood can be quite the mystery. In this article, I'm going to explain what the browser does to understand what it should show to the user.
```

Reply:
Closing, as this is just a shift of tone and not really a required change per grammar/spelling